zapcore/encoder.go

```diff
@@ -24,9 +24,28 @@ import (
 	"strings"
 	"time"
 
+	"github.com/fatih/color"
+
 	"go.uber.org/zap/buffer"
 )
 
+var (
+	// TODO(pedge): add a test to verify all levels covered? It would be nice
+	// to maintain a slice of all levels in level.go for testing, even if there
+	// is already _minLevel and _maxLevel
```

Comment:
Don't mind putting one in `testutils`, though whatever code deals with levels should handle unexpected levels too.